sections/04_study.md

```diff
@@ -82,7 +82,97 @@ use interesting network architectures to approach single-cell data.
+and techniques have been proposed that are classifier-independent
+[Ditzler,Ditzler].
+
+So, how have neural networks been of use?  One of the first techniques of
+genome binning used neural networks [Abe]. Neural networks can be slow, and
+so far only TAC-ELM [tac-elm] has utilized it to taxonomically classify
+massive amounts of metagenomic data.  Also, neural networks can fail to
+perform if there are not enough training examples, which is the case with
+taxonomic classification (since only ~10% of estimated species have been
+sequenced).  For “de novo” genome binning, Essinger et al. use ART, a neural
+network algorithm called Adaptive Resonance Theory, to cluster similar
+genomic fragments and showed that it has better performance than K-means; 
+however, other methods based on interpolated Markov models [Salzberg] have
+performed better.  Due to the complexity of the problem, neural networks have
+been applied more to gene annotation (e.g. Orphelia [Hoff]), which usually
+have plenty of training examples.  Representations (similar to Word2Vec [ref]
+in natural language processing) for protein family classification has been
+introduced and classified with a skip-gram neural network [Asgari]. 
+Recurrent neural networks show good performance for homology and protein
+family identification [Hochreiter, Sonderby].  Interestingly, Hochreiter, who
+invented Long Short Term Memory, delved into homology/protein family
+classification in 2007, and therefore, deep learning is deeply rooted in
+functional classification methods.
+
+Most neural networks are being used for short sequence->taxa/function
+classification, where there is a lot of data for training (and thus suitable
+for NNs).  And, as a short side note, recurrent neural networks are showing
+success for base-calling (and thus removing the large error in the
+measurement of a pore's current signal) for the relatively new Oxford
+Nanopore sequencer [Boza].  However, due to small nubmers of metagenomic
+samples in studies, neural network uses for classifying phenotype from
+microbial composition are just beginning.   A standard MLP was able to
+classify wound severity from microbial species present in the wound [Kizek,
+2015].  Recently, multi-layer, recurrent networks (and convolutional
+networks) have been applied to microbiome genotype-phenotype, with Ditzler et
+al. being the first to associate soil samples with pH level using multi-layer
+perceptrons, deep-belief networks, and recursive neural networks (RNNs) 
+[Ditzler] .  Besides classifying the samples appropriately, Ditzler shows
+that internal phylogenetic tree nodes inferred by the networks are
+appropriate features representing low/high pH, which can provide additional
+useful information and new features for future metagenomic sample comparison.
+ Also, an initial study has show promise of these networks for diagnosing
+disease [Faruqi].  However, deep neural networks are not ideal for such
```

Comment:
This is structured in a very problem-centric way. For this review, I wonder if organizing this section as something like
* introductory paragraph to metagenomics
* places where deep NNs have been successful [with some interpretation as to why]
* places where deep NNs have not been successful [again maybe with some why]
* places where deep NNs have not been applied, but you think they should be
would be the way to go. I think all of these materials are essentially in place, so this would involve moving things around and linking components together. @agitter - what's your intuition? I don't want to suggest organization that doesn't work for your section.